tests/dcerpc/dcerpc-dce-stub-data/test.rules

```diff
@@ -0,0 +1,5 @@
+alert tcp any any -> any any (msg:"DCE Iface test";flow:established,to_server;dcerpc.iface:afa8bd80-7d8a-11c9-bef4-08002b102989;sid:1;)
```

Comment:
Let's keep the rules for what they're supposed to do. Since this is not an iface test, and we already do the same test in `dce-iface-02`, let's skip this.